system/usbmsc/Kconfig

```diff
@@ -17,13 +17,20 @@ menuconfig SYSTEM_USBMSC
 if SYSTEM_USBMSC
 
 config SYSTEM_USBMSC_NLUNS
-	int "Number of LUNs"
+	int "Number of const LUNs"
 	default 1
 	---help---
 		Defines the number of logical units (LUNs) exported by the USB
 		storage driver.  Each LUN corresponds to one exported block driver
 		(or partition of a block driver).  May be 1, 2, or 3.  Default is 1.
 
+config SYSTEM_USBMSC_EXT_NLUNS
```

Comment:
why not reuse SYSTEM_USBMSC_NLUNS?

Reply:
> why not reuse SYSTEM_USBMSC_NLUNS?

The `SYSTEM_USBMSC_NLUNS` will be used by `SYSTEM_USBMSC_DEVPATH1`, `SYSTEM_USBMSC_DEVPATH2` and `SYSTEM_USBMSC_DEVPATH3` firstly. 
It means that we MUST set 3 const LUNs before we can specific LUNs at runtime.

Reply:
yes, but it could simplify the config